### ISSUE TITLE
Request changing ariaHideApp default to false

### ISSUE THREAD

**Issue by yncat** [reactjs/react-modal#776]
### Summary:
 1. When placing modals without ariaHideApp={false} property, screen-reader users cannot access the modal at all because the default value is true and it adds aria-hidden DOM property to the component. 

### Expected behavior:
Modal dialogs should not have aria-hidden by default so that they are visible by screen-readers.

### Link to example of issue:
The minimum example in readme.md can reproduce this issue.

### Additional notes:
I am a blind web UX designer.
Since this component is used in many services, we have thousands of inaccessible modal dialogs at the moment.
I tried to patch it myself, but for some reason, I wasn't able to pass some of the tests. 3 tests fail even when I run the test at the unchanged master branch, so I suspect that this is happening only for my environment.
All I want to be changed is just one line. true to false. That will help us a lot.

**Comment by yncat** [comment]
Resolved in #777
